test/unit/specs/vislib/visualizations/tile_map.js

```diff
@@ -94,6 +98,55 @@ define(function (require) {
           });
         });
 
+        describe('nearestFeature method', function () {
+          it('should return an object', function () {
+            vis.handler.charts.forEach(function (chart) {
+              var lat = (Math.random() * 180) - 90;
+              var lng = (Math.random() * 360) - 180;
+              var point = L.latLng(lat, lng);
+              var mapData = chart.chartData.geoJson;
+              expect(_.isObject(chart.nearestFeature(point, mapData))).to.be(true);
+              expect(chart.nearestFeature(point, mapData).type).to.be('Feature');
```

Comment:
Need to check the values of the feature so we know its returning the right one, not just that it returns any feature.